feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -316,21 +316,27 @@ func TestZeroMaster(t *testing.T) {
 		highID:     inId0,
 		handle:     clientConnection(t, dut),
 		deviceID:   deviceId,
-		wantFail:   true,
-		wantStatus: 3,
+		wantFail:   false,
+		wantWrite:  false,
+		wantRead:   true,
+		wantStatus: 0,
 	}
 	t.Run(test.desc, func(t *testing.T) {
 		resp, err := streamP4RTArb(&test)
-		if err == nil && test.wantFail {
-			t.Errorf("Zero ElectionID (0,0) is allowed: %v", err)
+		if err != nil && test.wantFail {
 			removeClient(test.handle)
-		} else {
-			t.Logf("Zero ElectionID (0,0) connection failed as expected: %v", err)
+			t.Errorf("Zero ElectionID (0,0) is rejected by P4RT server: %v", err)
```

Comment:
it doesn't make sense why there should be an error here. The if-condition says this is the case when there is an error but we are _expecting_ a failure.

Reply:
The `wantFail` variable is not required at all. There is only one test case and it is required to pass.

Reply:
i will remove the wantFail and the err in my next commit. Thanks

Reply:
this is done!